### DISCUSSION TITLE
Unable to add multiple generators of the same type?

### DISCUSSION THREAD

**Discussion by TabithaTheNeko** [null2264/CobbleGen#43] [Q&A] [answered]
I've been trying to have multiple generators that can generate ores when the block form of said ore is the block below the generated block, and I was wondering what I was doing wrong.

[cobblegen.txt](https://github.com/null2264/CobbleGen/files/12564038/cobblegen.txt)

**Comment by null2264** [comment]
![image](https://github.com/null2264/CobbleGen/assets/46041660/75a2d91a-f6da-4ee4-a4a6-e6e4b8a2ca0a)

It's probably caused by the extra double-quote on `"minecraft:quartz_block""`, it should be colon instead `"minecraft:quartz_block":`

**Reply by TabithaTheNeko** [reply, marked as answer]
yeah i had found that one earlier but it still didn't work. I did find this site https://json-5.com/ that let helped with the format and it started working correctly, there was a bunch of unnecessary commas.
